sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Operations/ServiceRestClient.cs

```diff
@@ -497,19 +579,21 @@ internal HttpMessage CreateAnalyzeReceiptAsyncRequest(bool? includeTextDetails,
             request.Headers.Add("Content-Type", "image/jpeg");
             request.Headers.Add("Content-Type", "image/png");
             request.Headers.Add("Content-Type", "image/tiff");
+            request.Content = RequestContent.Create(fileStream);
             return message;
         }
         /// <summary> Extract field text and semantic values from a given receipt document. The input document must be of one of the supported content types - &apos;application/pdf&apos;, &apos;image/jpeg&apos;, &apos;image/png&apos; or &apos;image/tiff&apos;. Alternatively, use &apos;application/json&apos; type to specify the location (Uri or local path) of the document to be analyzed. </summary>
         /// <param name="includeTextDetails"> Include text lines and element references in the result. </param>
         /// <param name="contentType"> Upload file type. </param>
+        /// <param name="fileStream"> .json, .pdf, .jpg, .png or .tiff type file stream. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public async ValueTask<ResponseWithHeaders<AnalyzeReceiptAsyncHeaders>> AnalyzeReceiptAsyncAsync(bool? includeTextDetails, ContentType? contentType, CancellationToken cancellationToken = default)
+        public async ValueTask<ResponseWithHeaders<AnalyzeReceiptAsyncHeaders>> AnalyzeReceiptAsyncAsync(bool? includeTextDetails, ContentType? contentType, Stream fileStream, CancellationToken cancellationToken = default)
```

Comment:
An overload that takes stream is also here but not fully correct because we add all content types to headers

Reply:
So don't use it yet, we'll fix it with the next update.